row.go

```diff
@@ -25,3 +25,17 @@ func (r *Row) AddCell() *Cell {
 	r.Sheet.maybeAddCol(len(r.Cells))
 	return cell
 }
+
+// AddStreamCell takes as input a StreamCell, creates a new Cell from it,
+// and appends the new cell to the row.
+func (r *Row) AddStreamCell(streamCell StreamCell) {
```

Comment:
What is the use case for this function? I think `Row`s are only used for normal files.

Reply:
This is indeed no longer necessary since we no longer add the first row when creating the headers. But it was used before.